custom_components/hubitat/hubitatmaker/const.py

```diff
@@ -33,40 +34,98 @@ class DeviceCapability(StrEnum):
     TEMPERATURE_MEASUREMENT = "TemperatureMeasurement"
     THERMOSTAT = "Thermostat"
     WINDOW_SHADE = "WindowShade"
+    WINDOW_BLIND = "WindowBlind"
     FAN_CONTROL = "FanControl"
+    CURRENT_METER = "CurrentMeter"
+    ACCELERATION_SENSOR = "AccelerationSensor"
+    BATTERY = "battery"
+    BULB = "Bulb"
+    CHIME = "Chime"
+    INDICATOR = "Indicator"
+    OUTLET = "Outlet"
+    RELAY_SWITCH = "RelaySwitch"
+    SHOCK_SENSOR = "ShockSensor"
+    SIGNAL_STRENGTH = "SignalStrength"
+    SLEEP_SENSOR = "SleepSensor"
+    SMOKE_DETECTOR = "SmokeDetector"
+    SOUND_PRESSURE_LEVEL = "SoundPressureLevel"
+    SOUND_SENSOR = "SoundSensor"
+    STEP_SENSOR = "StepSensor"
+    TAMPER_ALERT = "TamperAlert"
+    THERMOSTAT_COOLING_SETPOINT = "ThermostatCoolingSetpoint"
+    THERMOSTAT_FAN_MODE = "ThermostatFanMode"
+    THERMOSTAT_HEATING_SETPOINT = "ThermostatHeatingSetpoint"
+    THERMOSTAT_SETPOINT = "ThermostatSetpoint"
+    TIMED_SESSION = "TimedSession"
+    TOUCH_SENSOR = "TouchSensor"
+    UV = "UltravioletIndex"
+    VALVE = "Valve"
+    VIDEO_CAMERA = "VideoCamera"
+    VOLTAGE_MEASUREMENT = "VoltageMeasurement"
+    WATER_SENSOR = "WaterSensor"
+    CONSUMABLE = "consumable"
+    CHANGE_LEVEL = "ChangeLevel"
+    LIGHT_EFFECTS = "LightEffects"
+    MOMENTARY = "Momentary"
+    PUSH = "push"
+    SPEECH_SYNTHESIS = "SpeechSynthesis"
+    GAS_DETECTOR = "GasDetector"
+    AIR_QUALITY = "AirQuality"
+    LIQUID_FLOW_RATE = "LiquidFlowRate"
 
 
 class DeviceAttribute(StrEnum):
     ACCELERATION = "acceleration"
     ALARM = "alarm"
+    AQI = "aqi"
+    AIR_QUALITY_INDEX = "airQualityIndex"
+    AMPERAGE = "amperage"
     BATTERY = "battery"
     CARBON_DIOXIDE = "carbonDioxide"
+    CARBON_DIOXIDE_LEVEL = "carbonDioxide-Level"
     CARBON_MONOXIDE = "carbonMonoxide"
+    CARBON_MONOXIDE_LEVEL = "carbonMonoxide-Level"
     CODE_CHANGED = "codeChanged"
     CODE_LENGTH = "codeLength"
     COLOR_MODE = "colorMode"
     COLOR_NAME = "colorName"
     COLOR_TEMP = "colorTemperature"
     CURRENT = "current"
     CONTACT = "contact"
+    CUMULATIVE_LITER = "cumulative_liter"
+    CUMULATIVE_CUBIC_METER = "cumulative_cubic_meter"
+    DAY_EURO = "day_euro"
+    DAY_LITER = "day_liter"
+    DAY_CUBIC_METER = "day_cubic_meter"
     DOOR = "door"
     DOUBLE_TAPPED = "doubleTapped"
+    DEW_POINT = "DewPoint"
     ENERGY = "energy"
     ENERGY_SOURCE = "energySource"
     ENTRY_DELAY = "entryDelay"
     EXIT_DELAY = "exitDelay"
     HELD = "held"
+    HEAT_ALARM = "heatAlarm"
     HUE = "hue"
     HUMIDITY = "humidity"
+    HOME_HEALTH = "HomeHealth"
```

Comment:
homeHealth?

Reply:
Also not an official HE field, but its part of the driver for this device [Eurotronic Air Quality Sensor](https://eurotronic.org/wp-content/uploads/2021/12/Air-Quality-Sensor-Z-Wave-Plus_BDA-Web.pdf)